V13 7.0.3 alpha - XP Macro broken

```js
Error: An error occurred while rendering Dialog 192. Cannot convert undefined or null to object
    at Hooks.onError (foundry.mjs:23595:24)
    at foundry.mjs:36909:15Caused by: TypeError: Cannot convert undefined or null to object
    at Object.keys (<anonymous>)
    at Dialog.getData (foundry.mjs:118542:28)
    at Dialog._render (foundry.mjs:36952:29)
    at Dialog.render (foundry.mjs:36907:10)
    at showXP (pf2e.mjs:1553:286)
    at Dialog.callback (pf2e.mjs:1553:969)
    at Dialog.submit (foundry.mjs:118656:46)
    at #onClickButton (foundry.mjs:118585:10)
    at HTMLButtonElement.dispatch (jquery.min.js:2:40035)
    at v.handle (jquery.min.js:2:38006)
onError @ foundry.mjs:23596
```

![Image](https://github.com/user-attachments/assets/12fd5c12-2a64-4670-b5dd-b56c9625c89a)

--- a/src/scripts/macros/xp/dialog.ts
+++ b/src/scripts/macros/xp/dialog.ts
@@ -148,6 +148,7 @@ function showXP(partyLevel: number, partySize: number, npcLevels: number[], haza
     new foundry.appv1.api.Dialog({
         title: "XP",
         content: dialogTemplate(xp),
+        buttons: {},
     }).render(true);
 }
 

--- a/types/foundry/client/appv1/api/dialog-v1.d.mts
+++ b/types/foundry/client/appv1/api/dialog-v1.d.mts
@@ -180,7 +180,7 @@ export interface DialogData {
     title?: string;
     content?: string | HTMLElement | (() => string | HTMLElement);
     close?: (html: HTMLElement | JQuery) => void;
-    buttons?: Record<string, DialogButton>;
+    buttons: Record<string, DialogButton>;
     default?: string;
     render?: (html: HTMLElement | JQuery) => void;
 }

